crates/optimism/tests/devnets/opreth-seq-opgeth-val.yaml

```diff
@@ -0,0 +1,61 @@
+# A simple network configuration for kurtosis (https://github.com/ethpandaops/optimism-package)
```

Comment:
The filename seems a bit too much 😀. I think we can rename this as `sequencer-historical-proof.yaml` and `validator-historical-proof.yaml`